scripts/fixtures/eform/test-pattern.html

```
@@ -0,0 +1,605 @@
+</div>
+
+<div id="page2" class="test-page">
+  <div class="page-inner">
+    <section class="section">
+      <h2>Layout, Media, and Edge Cases</h2>
+      <div class="grid-2">
+        <label class="long-label">Long label wrapping: This label intentionally contains a very long unbroken token ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqrstuvwxyz so renderer regressions in clipping or overflow are visible.
+          <input type="text" name="test_pattern_long_label" id="test_pattern_long_label" value="Long label value">
+        </label>
+        <label>Conditional toggle
+          <span class="inline-options"><label><input type="checkbox" name="test_pattern_toggle_details" id="test_pattern_toggle_details" value="show" checked> Show detail panel</label></span>
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)